app/assets/stylesheets/base/main.scss

```diff
@@ -31,3 +31,8 @@ body {
   visibility: visible;
   font-size: 18px;
 }
+
+a {
+  color: var(--link-brand-color);
+  text-decoration: none;
+}
```

Comment:
Just out of curiosity, how come we had to move this from `/scaffolds.scss` into `/base/main.scss`? 👀

Reply:
So `/app/assets/stylesheets/*.scss` files are not part of Crayons. What is Crayons:

- `/app/assets/stylesheets/crayons.scss`
- `/app/assets/stylesheets/base/*`
- `/app/assets/stylesheets/components/*`
- `/app/assets/stylesheets/config/*`

Therefore that link styling wasn't really part of Crayons (likely one of the many missed things while building out crayons :D) so I had to move it.

Reply:
Ah, that's good to know! Thanks for explaining, @ludwiczakpawel  🙏